lib/cartopy/examples/gridliner.py

```diff
@@ -0,0 +1,40 @@
+"""
+Gridlines and tick labels
+-------------------------
+
+These examples demonstrate how to quickly add longitude
+and latitude gridlines and tick labels on a non-rectangular projection.
+
+As you can see on the first example,
+longitude labels may be drawn on left and right sides,
+and latitude labels may be drawn on bottom and top sides.
+Thanks to the ``dms`` keyword, minutes are used when appropriate
+to display fractions of degree.
+
+
+In the second example, labels are still drawn at the map edges
+despite its complexity, and some others are also drawn within the map
+boundary.
+
+"""
```

Comment:
Please add some `__tags__`.

Reply:
I added tags.

It should be approriate to have a predefined list of tags.
``"Lines and polygons"`` is often used in examples but is not clear of any user as for instance ``"Feature"``.